index.Rmd

```diff
@@ -505,135 +478,45 @@ LUTGewata
 This `data.frame` represents a lookup table for the raster we just loaded. The `$ID` column corresponds to the values taken on by the `lulc` raster, and the `$Class` column describes the LULC classes assigned. In `R` it is possible to add an attribute table to a raster. In order to do this, we need to coerce the raster values to a factor from an integer and add a raster attribute table.
 
 ```{r}
+# Set SpatRaster to categorical
 lulc <- as.factor(lulcGewata)
-# assign a raster attribute table (RAT)
+
+# Assign a raster attribute table (RAT)
```

Comment:
`terra` no longer uses RATs. It switched to cats. Yes, really ;) In `raster` they used to also have a function `ratify()`, which is now `catalyze()` ;)
```suggestion
# Assign categories to the raster values
```

The text above also should be adjusted accordingly.